libafl/src/feedbacks/new_hash_feedback.rs

```diff
@@ -148,26 +148,32 @@ impl<O, S> HasObserverName for NewHashFeedback<O, S> {
     }
 }
 
+/// Default capacity for the [`HashSet`] in [`NewHashFeedback`].
+///
+/// This is reasonably large on the assumption that you expect there to be many
+/// runs of the target, producing many different feedbacks.
+const DEFAULT_CAPACITY: usize = 4096;
+
 impl<O, S> NewHashFeedback<O, S>
 where
     O: ObserverWithHashField + Named + Debug,
 {
-    /// Returns a new [`NewHashFeedback`].
+    /// Returns a new [`NewHashFeedback`] with a capacity of 4096 `u64`s.
     /// Setting an observer name that doesn't exist would eventually trigger a panic.
     #[must_use]
     pub fn with_names(name: &str, observer_name: &str) -> Self {
         Self {
             name: name.to_string(),
             observer_name: observer_name.to_string(),
-            capacity: 0,
+            capacity: DEFAULT_CAPACITY,
             o_type: PhantomData,
         }
     }
 
-    /// Returns a new [`NewHashFeedback`].
+    /// Returns a new [`NewHashFeedback`] with a capacity of 4096 `u64`s.
```

Comment:
I wouldn't note the capacity here, it sound like it wouldn't work for more than 4096 entries, but it will simply reallocate.

Reply:
Ok, removed!

Reply:
Sweet thx :)